pkgs/by-name/vo/vocab/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  fetchFromGitHub,
+  lib,
+  python3Packages,
+  translate,
+}:
+python3Packages.buildPythonPackage {
```

Comment:
Python programs *outside of the Python package set* should use `buildPythonApplication` instead